[CUBRIDQA-831] update ctp_install_guide.md (#574)
CUBRID/cubrid-testtools: doc/ctp_install_guide.md

              ctp.sh medium medium    #execute medium twice
              ctp.sh webconsole start #start web console to view sql test results

## 2. Install CTP in Windows platform  
* ### Install Visual Studio 2017  
    Visual Studio is required by  CUBRID/bin/make_locale.bat.  
    When install Visual Studio 2017, choose `Workloads` tab, in `Windows (3)` section, choose `Desktop development with C++`, then click `Install` or `Modify` to start the installation.  

>     After installation, check system variable '%VS140COMNTOOLS%'   

Reviewer: ```suggestion
    After installation, check system variable `VS140COMNTOOLS`
```

  * If `VS140COMNTOOLS` is not added to the system variables automatically, please add it manually.  

>       VS140COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Tools\     

Reviewer: ```suggestion
      ```VS140COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Tools\ ```
```

* ### Install cygwin  
  * Required packages:  `wget`, `zip`, `unzip`, `dos2unix`, `bc`, `expect`.  
    Do not choose: `gcc` and `MinGW`  
  * Package versions required  
      Unsatisfied versions which will lead to many case failures:  

>       grep: 3.0-2  

Reviewer: Indent each item

      gawk: 4.1.4-3  
      sed: 4.4-1  
      Satisfied versions:  

>       In my test, I use:  

Reviewer: Remove 
`In my test, I use: `

      gawk: 4.1.3-1  
      grep: 3.0-1  
      sed: 4.2.2-3  

>       To install the old versions, please refer to this comment Install old packages of cygwin  

Reviewer: Which comment?

>       To install the old versions, please refer to this comment Install old packages of cygwin  

Reviewer: ```suggestion
      To install the old versions, please refer to this comment Install old packages of cygwin.
```

              ctp.sh medium medium    #execute medium twice
              ctp.sh webconsole start #start web console to view sql test results

## 2. Install CTP in Windows platform  
* ### Install Visual Studio 2017  

>     Visual studio is used by make_locale.bat  

Reviewer: ```suggestion
    Visual Studio is required by  CUBRID/bin/make_locale.bat.
```

>     When install visual studio 2017, choose 'Workloads' view(tab), in 'Windows (3)'section, choose "Desktop development with C++", then click 'Install' or 'Modify' to start the installation.  

Reviewer: Check all lower case "visual studio" to be upper for first letter.

>     When install visual studio 2017, choose 'Workloads' view(tab), in 'Windows (3)'section, choose "Desktop development with C++", then click 'Install' or 'Modify' to start the installation.  

Reviewer: ```suggestion
    When install Visual Studio 2017, choose `Workloads` tab, in `Windows (3)` section, choose `Desktop development with C++`, then click `Install` or `Modify` to start the installation.  
```

    After installation, check system variable '%VS140COMNTOOLS%'   

>   * If 'VS140COMNTOOLS' is not add to the system variables automatically, please add it manually.  

Reviewer: ```suggestion
  * If `VS140COMNTOOLS` is not added to the system variables automatically, please add it manually.  
```

>       Variable name: VS140COMNTOOLS  

Reviewer: Replace with: 

```
VS140COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Tools\
```

      Variable value: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Tools\     
* ### Install cygwin  

>   * We need choose this packages manually since they will not be installed by default:wget, zip, unzip, dos2unix, bc, expect.  

Reviewer: ```suggestion
  * Required packages:  `wget`, `zip`, `unzip`, `dos2unix`, `bc`, `expect`.
```

>       gcc and mingw packages do not need to be installed.  

Reviewer: ```suggestion
     Do not choose: `gcc` and `MinGW`
```

>   * Check the versions of these packages (or components): gawk, grep, sed  

Reviewer: ```suggestion
  * Package versions required
```

>       The invalid versions for cygwin components:  

Reviewer: ```suggestion
      Unsatisfied versions which will lead to many case failures:
```

      grep: 3.0-2  
      gawk: 4.1.4-3  
      sed: 4.4-1  

>       We must use the versions before the versions list above.  

Reviewer: ```suggestion
       Satisfied versions:
```

      In my test, I use:  
      gawk: 4.1.3-1  
      grep: 3.0-1  
      sed: 4.2.2-3  

>       To install the old versions. please refer to this comment Install old packages of cygwin  

Reviewer: ```suggestion
      To install the old versions, please refer to this comment Install old packages of cygwin  
```
And miss content?

  * Change environment variable 'PATH'  
    Add 'C:\cygwin64\bin' in the 'PATH' 
* ### Install git  
    Download git in https://git-for-windows.github.io/.  
    In the installation wizard, choose these options:  

>     'Adjusting your PATH environment', choose 'Use Git from the Windows Command Prompt'  

Reviewer: ```suggestion
    `Adjusting your PATH environment`, choose `Use Git from the Windows Command Prompt`  
```

>     'Confifuring the line ending conversions', choose 'Checkout as-is, commit as-is'  

Reviewer: ```suggestion
    `Confifuring the line ending conversions`, choose `Checkout as-is, commit as-is`  
```

              ctp.sh medium medium    #execute medium twice
              ctp.sh webconsole start #start web console to view sql test results

## 2. Install CTP in Windows platform  
* ### Install Visual Studio 2017  
    Visual Studio is required by  CUBRID/bin/make_locale.bat.  
    When install Visual Studio 2017, choose `Workloads` tab, in `Windows (3)` section, choose `Desktop development with C++`, then click `Install` or `Modify` to start the installation.  
    After installation, check system variable `VS140COMNTOOLS`   
  * If `VS140COMNTOOLS` is not added to the system variables automatically, please add it manually.  
    ```
    VS140COMNTOOLS=C:\Program Files (x86)\Microsoft VisualStudio\2017\Community\Common7\Tools\    
    ``` 
* ### Install cygwin  
  * Required packages:  `wget`, `zip`, `unzip`, `dos2unix`, `bc`, `expect`.  
    Do not choose: `gcc` and `MinGW`  
  * Package versions required  
    Unsatisfied versions which will lead to many case failures:  
    ```
    grep: 3.0-2  
    gawk: 4.1.4-3  
    sed: 4.4-1  
    ```
    Satisfied versions:  
    ```
    gawk: 4.1.3-1  
    grep: 3.0-1  
    sed: 4.2.2-3  
    ```

>     To install the old versions, please refer to this satisfied versions Install old packages of cygwin.  [Satisfied versions comment reference.](http://jira.cubrid.org/browse/CUBRIDQA-422?focusedCommentId=4747548&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4747548)

Reviewer: Add related information to this guide.
I don't think jira's content is easy to read. We need guide well formated.

              ctp.sh medium medium    #execute medium twice
              ctp.sh webconsole start #start web console to view sql test results

## 2. Install CTP in Windows platform

> * ### Install JDK  

Reviewer: Remove "Install JDK".
Since it is included in linux steps. You descibe to follow it.
